app/views/ops/_zone_form.html.haml

```diff
@@ -1,5 +1,7 @@
 - url = url_for_only_path(:action => 'zone_field_changed', :id => (@zone.id || "new"))
 - disabled_name = !@zone.name.nil? && !@edit[:current][:name].nil?
```

Comment:
Would it make more sense to rename the variable to "editing"?

And what about setting the variable in the controller? 

Also the condition seems a bit as if the empty name was an effect of editing so determining if editing is happening by this does not look very readable. Can you simplify the condition if you move it to the controller?

Reply:
What do you think now?

I changed the variable name to "editing" and moved the condition to the controller, where I set editing to true in editing branch of settings_replace_right_cell.